src/page/blog/article-page.jsx

```diff
@@ -232,22 +232,33 @@ export const Article = ({ type }) => {
   const articleJsx = (
     <>
       {!successful ? (
-        <div className="relative flex flex-row justify-center mx-auto -top-5 pt-20">
+        <div
+          className="relative flex flex-row justify-center mx-auto -top-5 pt-20"
+          data-test-id="article-page-button-delete"
+        >
           {type === "edited" && (
             <button
               onClick={openDeleteConfirmationModal}
               className=" absolute top-10 right-10 bg-transparent text-danger font-bold text-lg mt-10"
+              data-test-id="article-page-button-delete"
             >
               Delete
             </button>
           )}
-          <ToastContainer className={"absolute"} />
-          <Form className={"w-[700px] ml-[224px] p-5 bg-white"}>
+          <ToastContainer
+            className={"absolute"}
+            data-test-id="article-page-toast-container"
+          />
+          <Form
+            className={"w-[700px] ml-[224px] p-5 bg-white"}
+            data-test-id="article-page-form"
+          >
```

Comment:
Interesting enough, the green syntax highlighted divs do apply the data test id attribute, even if it's a component.